CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contributor guide
+
+First of all, thank you for taking the time to contribute!✌️ 🎉
+
+We love your input regardless of its size, content or scope - whether it is an issue, feature request, bug-fix or documentation enhancement.
+
+Thanks 🙏 You rock 🤘
+
+## Table Of Contents
+
+[Getting started](#getting-started-👶)
+
+[Code of Conduct](#code-of-conduct)
+
+[License](#license)
+
+[How to report a bug?](#how-to-report-a-bug-🐛)
+
+[How to request the change?](#how-to-request-the-change-💡)
+
+[How to submit a change?](#how-to-submit-a-change-🚩)
+
+[Style guide](#style-guide-👗)
+
+[Recognision model](#recognition-model-🧐)
+
+[Any questions?](#any-questions-❓)
+
+
+## Getting started 👶
+
+For an introduction, please read [`README`](README.md) file. You can also find vignettes [here](vignettes/). Most likely you would find a "Getting started" article. Please also see a package manual in form of `pkgdown` website.
+## Code of Conduct
+
+This project is governed by [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to follow the rules outlined there.
+
+## License
+
+All of your code changes would be under the same [license](LICENSE) that covers this project.
+
+## How to report a bug 🐛 ?
+
+We use GitHub to track issues, feature requests as well as bugs. Before opening a new issue please double-check if it's already reported (but don't worry if  to be a duplicate - we will manage). If already there - up-vote ⬆️. We will have a look at it faster!
+
+We kindly ask you to write a good issue with a minimal and reproducible example that showoff the problem. It's would be also great to know your local environment (R session info) as well. You will find our guides in the template while opening a bug ticket. This would speed up the fixing process a lot! 📈.
+
+## How to request the change 💡 ?
+
+In case you find a missing feature or you have an idea how to enhance existing functionalities - please let us know by creating a new issue. Before opening a new issue please double-check if it's already reported (but don't worry if occurs to be a duplicate - we will manage). If already there - up-vote ⬆️. We will have a look at it faster!
+
+Please elaborate on "why?" - what's the context, what's the benefit and for whom it is. You will find our guides in the template while opening a feature request ticket. This would help us prioritize and submit meaningful changes.
```

Comment:
```suggestion
Please elaborate on "why?" - what's the context, what's the benefit and for whom. You will find our template when opening a feature request ticket. This will help us prioritize and submit meaningful changes.
```